mods/ts/rules/aircraft.yaml

```diff
@@ -236,6 +239,7 @@ APACHE:
 		Cost: 1000
 	Tooltip:
 		Name: Harpy
+		Description: Support gunship.\n  Strong vs Infantry, Light Vehicles, Aircraft\n  Weak vs Buildings, Tanks
```

Comment:
Tanks &rarr; Vehicles?  TS only has a couple of tanks, and I don't think any of them are AA?